site/src/pages/TemplateVariablesPage/TemplateVariablesPage.tsx

```diff
@@ -0,0 +1,107 @@
+import { useMachine } from "@xstate/react"
+import {
+  CreateTemplateVersionRequest,
+  TemplateVersionVariable,
+  VariableValue,
+} from "api/typesGenerated"
+import { useOrganizationId } from "hooks/useOrganizationId"
+import { FC } from "react"
+import { Helmet } from "react-helmet-async"
+import { useTranslation } from "react-i18next"
+import { useNavigate, useParams } from "react-router-dom"
+import { templateVariablesMachine } from "xServices/template/templateVariablesXService"
+import { pageTitle } from "../../util/page"
+import { TemplateVariablesPageView } from "./TemplateVariablesPageView"
+
+export const TemplateVariablesPage: FC = () => {
+  const { template: templateName } = useParams() as {
+    organization: string
+    template: string
+  }
+  const organizationId = useOrganizationId()
+  const navigate = useNavigate()
+  const [state, send] = useMachine(templateVariablesMachine, {
```

Comment:
I said that previously but maybe would be interesting to use `react-query` but no big deal.

Reply:
I think that I will leave it for further improvement as I'd like to deliver this PR as is, as it just works. I will try to frame more time for `react-query` to play with it.